src/lib/Handset/handset.h

```diff
@@ -95,6 +95,11 @@ class Handset
      */
     uint32_t GetRCdataLastRecv() const { return RCdataLastRecv; }
 
+    /**
+     * @brief Can be used to force a connected callback for debugging
+     */
+    void forceConnection() { if (connected) connected(); }
+
```

Comment:
I'd rather not introduce a public API for a debugging option that should remain inside the handset code.
So let's just put the body in the `#ifdef` in `devHandset.cpp`

Reply:
i agree in principle, but `connected` is [protected in Handset](https://github.com/ExpressLRS/ExpressLRS/pull/2823/files#diff-bd079ecb09409e2cb303c932a28cd6540a9769cea0c8a6ddc8b909a8a0209b3fR109), making that public seemed worse to me.

Reply:
Ah, right! devHandset does not extend handset. In that case how about wrapping the `forceConnection` in the ifdef as well then.

Reply:
done!